backend-config/types.go

```diff
@@ -109,8 +109,8 @@ type ConfigT struct {
 	UpdatedAt          time.Time                    `json:"updatedAt"`
 	Credentials        map[string]Credential        `json:"credentials"`
 	Connections        map[string]Connection        `json:"connections"`
-	Accounts           []Account                    `json:"accounts"`
-	AccountDefinitions []AccountDefinition          `json:"accountDefinitions"`
+	Accounts           map[string]Account           `json:"accounts"`
```

Comment:
let's verify config schema compatibility in a dev environment before change in config backend goes live